osv/types.go

```diff
@@ -0,0 +1,50 @@
+package osv
+
+type OsvAffected struct {
+	Package  *OsvPackage `json:"package,omitempty"`
+	Ranges   []OsvRange  `json:"ranges,omitempty"`
+	Versions []string    `json:"versions,omitempty"`
+
+	// Ecosystem *OsvEcosystem `json:"ecosystem_specific,omitempty"` //The meaning of the values within the object is entirely defined by the ecosystem
```

Comment:
You can use `interface{}`

```suggestion
	EcosystemSpecific interface{} `json:"ecosystem_specific,omitempty"`
```

Reply:
Done